bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -634,6 +646,65 @@ attributes from the config_cache.xml file (with keys converted to upper-case).
 <!-- ==================================
      For an exact match for these grids
 ======================================== -->
+<!--
+     clm5_0 physics with CAM7.0 grids...
```

Comment:
Just to confirm my suspicion:
- Is this section an identical copy of the corresponding "clm5_0 with CAM6.0" section?
- Is the same true throughout namelist_defaults_ctsm.xml, i.e. to introduce the CAM7 lines, you copied the CAM6 lines? Or are there exceptions to that rule?
- Sections that are identical copies for which I suggested changes, my suggestions likely apply to the preexisting lines, as well.

Reply:
Yes, that's correct this is copying the previous section for CAM6.0 for CAM7.0. There's a lot of evil code duplication here that longer term we should handle better.

And yes to add the new lines I searched for cam6.0 and added duplicate lines for cam7.0.

Good point about changes applying to both.

Reply:
I tried to add comments to all sections. So I think I've handled this.